tools/regression_test.py

```diff
@@ -519,6 +520,53 @@ def get_info_from_log_file(info_type: str, log_path: Path,
     return info_value
 
 
+def run_cmd(cmd_lines: List[str], log_path: Path):
+    """
+    Args:
+        cmd_lines: (list[str]): A command in multiple line style.
+        log_path (Path): Path to log file.
+
+    Returns:
+        int: error code.
+    """
+    import platform
+    system = platform.system().lower()
+
+    if system == 'windows':
+        sep = r'`'
+    else:  # 'Linux', 'Darwin'
+        sep = '\\'
+    cmd_for_run = ' '.join(cmd_lines)
+    cmd_for_log = f' {sep}\n'.join(cmd_lines) + '\n'
+    parent_path = log_path.parent
+    if not parent_path.exists():
+        parent_path.mkdir(parents=True, exist_ok=True)
```

Comment:
why not using `mmcv.mkdir_or_exist`?

Reply:
Because it's the instance of `Path` class, not of `str` type.

Reply:
but it should contains the string. Anyway, codes can be kept.